### PR TITLE
Vault fixing

### PR DESCRIPTION
**Pull request by Kirandeep-Singh-Khehra**
### Description of the change
Changes the pull repo of vault from `docker.io/library` to `docker.io/hashicorp`
For `tests_required.py` increased the request per page to 100(max api can go) from 30(dafault)

### CODE REVIEW

File: .github/workflows/readme_updater.yml [hunk 1]
```diff
@@ -43,631 +43,631 @@ jobs:
...
+        uses: peter-evans/dockerhub-description@v3
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
           repository: rapidfort/airflow
           readme-filepath: ./community_images/airflow/airflow/bitnami/README.md.dockerhub
           short-description: RapidFort optimized, hardened image for Apache Airflow
       - name: Docker Hub Description Apache Airflow Ironbank
-        uses: peter-evans/dockerhub-description@v4
+        uses: peter-evans/dockerhub-description@v3
```
Comment (AnmolVirdi) [inline .github/workflows/readme_updater.yml:54]: why have we downgraded from v4 to v3?

File: community_images/haproxy/ironbank/image.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ input_registry:
   account: ironbank
 repo_sets:
   - opensource/haproxy/haproxy24:
-      input_base_tag: "v2.4."
+      input_base_tag: "2.4."
```
Comment (AnmolVirdi) [inline community_images/haproxy/ironbank/image.yml:24]: Are we sure about this?

**Review by AnmolVirdi — CHANGES_REQUESTED** [review]

File: .github/workflows/readme_updater.yml (hunk 1 above)
Comment (Kirandeep-Singh-Khehra) [reply .github/workflows/readme_updater.yml:54]: This is some bug in execution path of `image_generator.sh`. It lowers this version. But some bot raises PR which is changing it back to v4. Ill see to it and fix it if possible.

File: community_images/haproxy/ironbank/image.yml (hunk 2 above)
Comment (Kirandeep-Singh-Khehra) [reply community_images/haproxy/ironbank/image.yml:24]: This is automatically updated by `prepare_ironbank_tags.sh` and the repo1 also have made same changes. i.e. removed `v` from tag and latest tag is `2.4.25`. So, its a good fix.

**Pull request merged**